docs/src/index.md

```diff
@@ -36,3 +36,8 @@ sinkhorn_unbalanced2
 ```@docs
 quadreg
 ```
+
+## Auxiliary functions
+```@docs
+discretemeasure
+```
```

Comment:
Maybe we could not export the function until it is actually used in some examples (e.g. in the divergence PR)? This would make it easier to change the implementation and functionality if desired - if it is exported we can't remove it and only modify it in non-breaking ways if we don't want to make a breaking release.

```suggestion
```